waagent

```diff
@@ -5876,7 +5880,7 @@ def main():
     if len(sys.argv) == 1:
         sys.exit(Usage())
 
-    LoggerInit('/var/log/waagent.log','/dev/console')
+    LoggerInit('/dev/stdout',None)
```

Comment:
They wouldn't, because: 
- the log is still being written to stdout
- the hardcoded value `/var/log/waagent.log` was just migrated to waagent.conf which is a part of this software
- the change is documented in the README